doc/reference/reference_sql/sql_statements_and_clauses.rst

```diff
@@ -485,8 +485,9 @@ Data types may also appear in :ref:`CAST <sql_function_cast>` functions.
     column7 STRING, column8 STRING COLLATE "unicode",
     column9 TEXT, columna TEXT COLLATE "unicode_sv_s1",
     columnb VARCHAR(0), columnc VARCHAR(100000) COLLATE "binary",
-    columnd VARBINARY,
-    columne SCALAR, columnf SCALAR COLLATE "unicode_uk_s2");
+    columnd UUID,
```

Comment:
It is not added as a non-last attribute. The text that I see in the changed file looks like this:
   -- with all possible data types and aliases
   CREATE TABLE t
   (column1 BOOLEAN, column2 BOOL,
    column3 INT PRIMARY KEY, column4 INTEGER,
    column5 DOUBLE,
    column6 NUMBER,
    column7 STRING, column8 STRING COLLATE "unicode",
    column9 TEXT, columna TEXT COLLATE "unicode_sv_s1",
    columnb VARCHAR(0), columnc VARCHAR(100000) COLLATE "binary",
    columnd UUID,
    columne VARBINARY,
    columnf SCALAR, columng SCALAR COLLATE "unicode_uk_s2");

Reply:
Yes, I see. And the question is why did you change `collumnd` type to UUID, instead of making the new attribute `collumng` of UUID type? Anyway, I am totally OK with this change, just trying to get your point regarding it.

Reply:
So I misunderstood your question, sorry. For this example we can put the columns in any order. I was not trying to make a point. I am wondering now: is there any way that putting columns in a different order affects performance? But, regardless what the answer for that is, it won't matter for this example.